app/views/pages/webhooks.html.md

```diff
@@ -0,0 +1,109 @@
+# Webhook Notifications
+
+## 1. Introduction
+Webhook notifications allow suppliers to subscribe to the PECS API system and receive notifications of when a move record is created or updated. The notification is transmitted via an HTTP POST to an endpoint specified by the supplier. The notification is signed with a shared key to ensure the authenticity of the message. If the endpoint is unavailable then the PECS system will re-attempt the delivery later, retrying up to 25 times.
+
+
+## 2. Subscribing to notifications
+The subscription process is currently manual - please contact the Book a Secure Move via your Slack channel with the following details:
+
+* Supplier name
+* Callback URL: this should be an `https://` endpoint which is publicly accessible and should not require authentication
+* Secret: this is a shared secret which is used to generate a SHA-256 HMAC signature to guarantee the authenticity of the notification
+
+Technical note for the Book a Secure Move team to manually create the subscription record:
+
+* connect to the Staging kubernetes pod and in a rails console, run:
+
+        supplier = Supplier.find_by(key: "<SUPPLIER-KEY>")
+        supplier.subscriptions << Subscription.new(callback_url: "<CALLBACK-URL>", secret: "<SECRET>")
+
+Once the Book a Secure Move team have actioned the request, notifications of moves events will be immediately sent to the specified `callback_url`.
+
+
+## 3. Receiving notifications
+The supplier should ensure that the designated endpoint is available to receive notifications at all times. If for any reason the endpoint is offline when a notification is attempted, the PECS system will retry the notification later with an increasing random exponential delay. The system will continue to retry the notification up to 25 times. On the first day the notification will be attempted approximately 14 times (see [this table](https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry) for the approximate schedule).
```

Comment:
Would it be worth adding another section which details what happens on failure. That could include the 25-retry text above, but also what happens once that limit is reached – i.e. do we discard the notification or take the webhook into an error state?